Resources/Prototypes/Entities/Objects/Misc/business card.yml

```diff
@@ -0,0 +1,42 @@
+- type: entity
+  id: SyndicateBusinessCard
+  name: Syndicate business card
+  parent: BaseItem
+  description: A black card with the syndicate's logo. There's something written on the back
+  components:
+    - type: Sprite
+      sprite: Objects/Misc/Business_сard.rsi
+      state: Synicate card
```

Comment:
```suggestion
      state: Synicate_card
```
You forgot to fix the name of the state in the prototype.
![image](https://github.com/user-attachments/assets/b0925491-0540-45d6-b110-2576760e930d)